components/app.js

```diff
@@ -74,10 +80,35 @@ class App {
     };
     $.ajax("https://sgt.lfzprototypes.com/api/grades/" + id, appConfig)
   }
+
   handleDeleteGradeError(error) {
     console.error();
   }
   handleDeleteGradeSuccess() {
     this.getGrades();
   }
+
+  handleEditGradeError(error) {
+    console.error();
+  }
+  handleEditGradeSuccess() {
+    this.getGrades();
+  }
+
+  editGrade(id, name, course, grade) {
+    var appConfig = {
+      type: "PATCH",
+      headers: {
+        "X-Access-Token": "jpVhjpBr"
+      },
+      success: this.handleEditGradeSuccess,
+      error: this.handleEditGradeError,
+      data: {
+        name: name,
+        course: course,
+        grade: grade
+      },
+    };
+    $.ajax("https://sgt.lfzprototypes.com/api/grades/" + id, appConfig)
+  }
```

Comment:
One optional improvement would be to only send the fields which have changed to avoid the extra network bandwidth.